projects/kit/directives/connected/connected.style.less

```diff
@@ -0,0 +1,40 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiConnected] {
+    --tui-icon-height: 2rem;
+    --tui-icon-width: 2rem;
+    --tui-left-offset: 1rem;
```

Comment:
Left offset is coming from padding on cells. How about we add `padding-inline-start: inherit` instead so it would be automatic. We can use `background-clip` so that padding is not filled with background color.

Reply:
Okay, I am trying
It's left padding in cell and zero padding in stepper

Reply:
it's magic working and calc padding automatic with `padding-inline-start`!
<img width="374" alt="image" src="https://github.com/user-attachments/assets/c3f6bf0d-db4c-454d-a09b-da00cbe72ea0">